data/themes/darktable.css

```diff
@@ -1424,8 +1424,15 @@ cell:selected
 /* Module in darkroom left panel */
 #color-picker-area
 {
-  min-height: 9em;
-  min-width: 9em;
+  min-height: 7em;
+  min-width: 7em;
```

Comment:
Set to 8em on those 2 lines.

Reply:
I'm not so sure about this one. See my comments below